docs/internals/howto-release-django.txt

```diff
@@ -188,8 +188,12 @@ A week before a security release
 A few days before any release
 -----------------------------
 
-#. As the release approaches, watch Trac to make sure no release blockers
-   are left for the upcoming release.
+#. As the release approaches, watch Trac to make sure no release blockers are
+   left for the upcoming release. Under exceptional circumstances, such as to
+   meet a pre-determined security release date, a release could still go ahead
+   with an open release blocker. The releaser is trusted with the decision to
+   release with an open release blocker and to postpone the release date of a
```

Comment:
Unless I'm reading this wrong, shouldn't this be an "or"? I mean, given an announced release date, and an unsolved release blocker, a releaser can release without the fix OR postpone the release?
```suggestion
   release with an open release blocker or to postpone the release date of a
```

Reply:
Agreed, I was getting caught up on this. I think it may be better to rely on the prior sentence about releasing with an blocker and this sentence cover the possibility of postponing a non-security release.